change v1alpha1 from capi controlplane and minor changes (#504)
suse-edge/suse-edge.github.io: asciidoc/product/atip-management-cluster.adoc

[#mgmt-cluster-image-definition-file-airgap]
==== Modifications in the definition file

> The `mgmt-cluster.yaml` file must be modified to include the `embeddedArtifactRegistry` section with the `images` field set to all container images to be included into the EIB output image. The `images` field must contain the list of all container images to be included in the output image.

Reviewer: The two sentences include duplicated information. I suggest something like the following:

```suggestion
The `mgmt-cluster.yaml` file must be modified to include the `embeddedArtifactRegistry` section.
In this section the `images` field must contain the list of all container images to be included in the output image.
```

Author: makes sense! changed! Thanks

[NOTE]
====

> The following is only an example of the `mgmt-cluster.yaml` file with the `embeddedArtifactRegistry` section included. Please take care of the image version depending on the components needed.

Reviewer: Not sure I fully understand the second sentence. How about;

```suggestion
The following is an example of the `mgmt-cluster.yaml` file with the `embeddedArtifactRegistry` section included.
Make sure to the listed images contain the component versions you need.
```

Author: much better! Thanks

Reviewer: Actually I added a superfluous  "to". Should be: Make sure the listed images contain the component versions you need.

Sorry.
